### PR TITLE
Create users and groups - Sync Endpoint

### PR DESCRIPTION
**Pull request by ShristiJain**
#### What is included in this PR?
This PR makes some amendments in the "create users" page of Sync - Endpoint. The creating users instructions were written twice and hence it was merged into one.

### CODE REVIEW

**Review by elmps2018 — APPROVED** [review]
This looks good to me! I think you just moved content around, right? Nothing got really added or deleted?

**Comment by ShristiJain** [comment]
@elmps2018 Yeah, I have merged the two headings and just deleted some repeated lines.

File: odkx-src/sync-endpoint-user-instructions.rst [hunk 1]
```diff
@@ -61,50 +99,3 @@ Assigning users to groups
 
     a. Navigate to the :guilabel:`memberUid` section.
     b. Click modify group members to manage members.
-    
-.. _sync-endpoint-setup-create-user:
```
Comment (wbrunette) [inline odkx-src/sync-endpoint-user-instructions.rst:65]: This link no longer exists.

File: odkx-src/sync-endpoint-user-instructions.rst [hunk 2]
```diff
@@ -15,11 +15,49 @@ Creating users
...
+      :width: 600
+
+  4. Fill out information for the new user and “create object.” Assign it to the *default_prefix_synchronize_tables* group. Will need to commit (confirm) that you want to create this entry on the next screen.
+
+    .. image:: /img/setup-create-user/setup-user4.png
+      :width: 600
+
+    We have now created the user! We just need to add the user to the respective group from the group settings.
+
+  5. Click the :guilabel:`+` sign next **ou=groups** to expand it. Within the unfolded menu, in the **ou=default_prefix** section, click on :guilabel:`gidNumber=503`, which is the group ID that corresponds to *default_prefix_synchronize_tables*. Groups correspond to the access permissions available to a certain user.
```
Comment (wbrunette) [inline odkx-src/sync-endpoint-user-instructions.rst:43]: This should have  subheading assigning the user a group

**Review by wbrunette — CHANGES_REQUESTED** [review]
This does not appear to have the structure we discussed. The top level in the side bar should be Users and Groups, and then 3 subsections that expand 1) Create a User 2) Create a Group, and 3) Assign a User to a group.

There should also be introductory paragraph that a user must be assigned to an ODK-X group to gain permissions to access the system. This should have a link to the Data Permission Filters
https://docs.odk-x.org/data-permission-filters/
so people can understand which groups do what.

File: odkx-src/sync-endpoint-user-instructions.rst [hunk 3]
```diff
@@ -7,25 +7,67 @@
   subdomain
 
 
+Users and Groups
+=================
+
+A user needs to be assigned to an ODK-X group to gain permissions to access the system. More information about groups and the roles is available in :ref:`Data Permission Filters<data-permission-filters>` section.
```
Comment (elmps2018) [inline odkx-src/sync-endpoint-user-instructions.rst:13]: ODK-X group and access the system are a bit vague here. Which system(s)? Is there more specific language that can be used? Maybe something more like "A user needs to be assigned to a group within ODK-X sync-endpoint to set their permissions and roles for ODK-X apps."

Comment (ShristiJain) [reply odkx-src/sync-endpoint-user-instructions.rst:13]: Yes Okay. I will modify the line.

**Comment by ShristiJain** [comment]
> 

Yes, I have made the changes that @wbrunette mentioned.

**Review by wbrunette — DISMISSED** [review]

**Comment by wbrunette** [comment]
@ShristiJain here is an outstanding PR that is an addition in your text. Could you add the note in and revise the text as @elmps2018 suggests? It would be good to get this section fixed all at once.
https://github.com/odk-x/docs/pull/189

File: odkx-src/sync-endpoint-user-instructions.rst [hunk 4]
```diff
@@ -44,67 +67,42 @@ Creating groups
...
 
-  3. Then, select the :guilabel:`Generic: User Account` template.
+  5. A few groups are created when the LDAP server is brought up, refer to :doc:`data-permission-filters` for descriptions of these groups.
 
-    .. image:: /img/setup-create-user/setup-user3.png
-      :width: 600
+   .. note::
 
-  4. Fill out information for the new user and “create object.” Assign it to the *default_prefix_synchronize_tables* group. Will need to commit (confirm) that you want to create this entry on the next screen.
+     A user needs to be assigned one of the roles in addition to any other group of your choosing. These roles are available as groups 500 (Row owner), 501 (Group read only), 502 (Group modify), 503 (Group privileged).
```
Comment (wbrunette) [inline odkx-src/sync-endpoint-user-instructions.rst:84]: This is incorrect. 500 is role SITE_ADMIN, 501 is ADMINISTER_TABLES, 502 is SUPER_USER_TABLES, 503 is SYNCHRONIZE_TABLES

**Review by wbrunette — CHANGES_REQUESTED** [review]

File: odkx-src/sync-endpoint-user-instructions.rst [hunk 5]
```diff
@@ -7,25 +7,48 @@
   subdomain
 
 
+Users and Groups
+=================
 
+A user needs to be assigned to a group within ODK-X sync-endpoint to set their permissions and roles for ODK-X apps. More information about groups and the roles is available in :ref:`Data Permission Filters<data-permission-filters>` section.
```
Comment (elmps2018) [inline odkx-src/sync-endpoint-user-instructions.rst:13]: Sync Endpoint should be in caps and I don't think it should have a hyphen. No "the" before roles.

File: odkx-src/sync-endpoint-user-instructions.rst [hunk 6]
```diff
@@ -7,25 +7,48 @@
...
+
+    .. image:: /img/setup-create-user/setup-user2.png
+      :width: 600
+
+  3. Then, select the :guilabel:`Generic: User Account` template.
+
+    .. image:: /img/setup-create-user/setup-user3.png
+      :width: 600
+
+  4. Fill out information for the new user and “create object.” Assign it to the *default_prefix_synchronize_tables* group. Will need to commit (confirm) that you want to create this entry on the next screen.
```
Comment (elmps2018) [inline odkx-src/sync-endpoint-user-instructions.rst:40]: Should be "You will need" not just "Will need"

**Review by elmps2018 — APPROVED** [review]
This looks good to me but @wbrunette or @EstherPhebe or @matt9j should check the technical details!

**Review by wbrunette — APPROVED** [review]

**Comment by aasthaa-007** [comment]
Hi @wbrunette 
I would like to fix this issue 
Can I proceed?

**Pull request merged**